smartmodule/examples/filter_error/src/lib.rs

```diff
@@ -0,0 +1,7 @@
+use fluvio_smartmodule::{Record, Result, smartmodule};
+
+#[smartmodule(filter)]
+pub fn filter(_: &Record) -> Result<bool> {
+    let _a: u32 = "dasdas".parse()?;
+    Ok(true)
+}
```

Comment:
`smartmodule/examples` doesn't seem like the best place so let me know if there is a better location.